setup.py

```diff
@@ -54,7 +54,7 @@ def get_version(rel_path):
 }
 
 EXTRA_REQUIRES["dev"] = (
-    EXTRA_REQUIRES["test"] + EXTRA_REQUIRES["docs"] + ["pylint", "black", "pre-commit"]
+    EXTRA_REQUIRES["test"] + EXTRA_REQUIRES["docs"] + ["pylint", "black", "pre-commit", "mypy"]
```

Comment:
Should this be included in 'test' instead of 'dev', since I'm assuming we'll want some sort of check including type hints to run as part of the test suite (say, on GitHub actions)?

Reply:
Yes. This should probably go into test eventually. While I'm implementing the changes, I thought I'd set it in the dev requirements.